client/components/empty-content/index.jsx

```diff
@@ -59,6 +64,8 @@ class EmptyContent extends Component {
 				<a
 					className="empty-content__action button is-primary"
 					href={ this.props.actionURL }
+					onMouseEnter={ this.props.actionHoverCallback }
+					onTouchStart={ this.props.actionHoverCallback }
```

Comment:
I've included this [`touchstart` event](https://developer.mozilla.org/en-US/docs/Web/Events/touchstart) to be consistent with what we've been doing in [other parts of our code](https://github.com/Automattic/wp-calypso/blob/6300bcfa7aae99ff2f0a349af74313bc9029bfa7/client/blocks/daily-post-button/index.jsx#L163-L164).